src/providers/elixir.rs

```diff
@@ -51,9 +45,6 @@ impl Provider for ElixirProvider {
             build_phase.add_cmd("mix assets.deploy".to_string());
         }
 
-        if mix_exs_content.contains("postgrex") && mix_exs_content.contains("ecto") {
-            build_phase.add_cmd("mix ecto.setup");
```

Comment:
Why was this removed?

Reply:
Ah, for me I didn't want this. Will add it back in.

The risk with running ecto.setup is it generally requires a DB connection, which is not available when the image is being built.